dolfyn/io/api.py

```diff
@@ -6,17 +6,41 @@
 from .hdf5 import load
 
 
-def read(fname, userdata=True, nens=None):
+def read(fname, userdata=True, nens=None, keep_orient_raw=False):
     """Read a binary Nortek (e.g., .VEC, .wpr, .ad2cp, etc.) or RDI
     (.000, .PD0, etc.) data file.
 
-    Notes
-    =====
-    This function will loop over binary readers until it finds one that works.
+    Parameters
+    ----------
+    filename : string
+               Filename of Nortek file to read.
+
+    userdata : True, False, or string of userdata.json filename
+               (default ``True``) Whether to read the
+               '<base-filename>.userdata.json' file.
+
+    nens : None (default: read entire file), int, or
+           2-element tuple (start, stop)
+              Number of pings to read from the file
+
+    keep_orient_raw : bool (default: False)
+        If this is set to True, the raw orientation heading/pitch/roll
+        data is retained in the returned data structure in the
+        ``dat['orient']['raw']`` data group. This data is exactly as
+        it was found in the binary data file, and obeys the instrument
+        manufacturers definitions not DOLfYN's.
```

Comment:
This is in the Heading, Pitch, Roll documentation:
"Also note that
``dat['orient']['raw']['heading']`` is unaffected by setting declination
as described in the next section."
Do you want to add similar statement here, too?